ETBezierHandle.h

```diff
@@ -40,9 +41,11 @@
 {
 }
 
-- (id) initWithManipulatedObject: (id)aTarget;
-- (id) initWithActionHandler: (ETActionHandler *)anHandler 
-           manipulatedObject: (id)aTarget;
+- (id) initWithManipulatedObject: (id)aTarget
```

Comment:
ETBezierEditingFreeLayout implementation needs to be updated to use -initWithManipulatedObject:objectGraphContext: to eliminate the following warning:

Etoile/Frameworks/EtoilePaint/ETBezierEditingFreeLayout.m:26:72: Instance method '-initWithManipulatedObject:' not found (return type defaults to 'id')